DigitalLearningSolutions.Web/Views/MyAccount/Index.cshtml

```diff
@@ -82,14 +83,32 @@
             </dd>
           </div>
 
-          <div class="nhsuk-summary-list__row account-summarylist_row">
+          <div class="nhsuk-summary-list__row account-detailslist_row">
             <dt class="nhsuk-summary-list__key">
               Email address:
             </dt>
             <dd class="nhsuk-summary-list__value">
               @Model.User
             </dd>
           </div>
+
+          <div class="nhsuk-summary-list__row account-detailslist_row">
+            <dt class="nhsuk-summary-list__key profile-picture__label">
+              Profile picture (optional):
+            </dt>
+            <dd class="nhsuk-summary-list__value">
+              <div class="profile-picture__image-box">
+                @if (Model.ProfilePicture != null)
+                {
+                  <img class="profile-picture__image" src="data:image;base64,@Convert.ToBase64String(Model.ProfilePicture)" alt="Profile Picture" />
+                }
+                else
+                {
+                  <img class="profile-picture__image" src="@Url.Content("~/images/avatar.png")" alt="Default Profile Picture" />

```

Comment:
Just a thought and I'm not 100% sure, but could "Default profile picture" indicate that this will be the user's profile picture if they don't upload anything? Would it be worth naming it "No profile picture"?

Reply:
Unsure, I guess it depends how the rest of the site and old behaves when a user doesn't have a set profile picture, do they always display with the default? Do we not display a profile picture? I went with default as it implied to me it was a non-user set profile picture.